### PR TITLE
added write review functionality

### PR DESCRIPTION
**Pull request by jfleming9357**

### CODE REVIEW

**Review by mattBman23 — APPROVED** [review]
Looks good. I recommend having spaces between each function to make it more readable, also is jackson11 like your ign or something?

**Comment by jfleming9357** [comment]
> Looks good. I recommend having spaces between each function to make it more readable, also is jackson11 like your ign or something?

Its the placeholder they used in the BRD

File: client/components/RatingsReviews/handleReviewClicks.jsx [hunk 1]
```diff
@@ -22,4 +22,16 @@ export const reportClick = (e) => {
   //   .catch((err) => {
   //     console.log(err);
   //   });
+};
+
+export const submitReview = (review, setReviews, reviews) => {
+  let url = 'proxy/api/fec2/hratx/reviews/';
+  review = JSON.parse(JSON.stringify(review));
```
Comment (daniel-he-dev) [inline client/components/RatingsReviews/handleReviewClicks.jsx:29]: just, why

**Review by daniel-he-dev — COMMENTED** [review]
get rid of the one line i commented on, then good to merge

File: client/components/RatingsReviews/handleReviewClicks.jsx (hunk 1 above)
Comment (jfleming9357) [reply client/components/RatingsReviews/handleReviewClicks.jsx:29]: oh right it was one of my desperate bug fixing attempts. sorry for any bleeding eyeballs

**Pull request merged**